### PR TITLE
Magiclysm - Updated the conjured weapons to be magic focus items

### PR DESCRIPTION
**Pull request by foulman**
#### Summary
```SUMMARY: Mods "Magiclysm - Conjured weapons corrected to possess magic_focus flag"```

#### Purpose of change
Spellcaster items should be magic focus items by default as they are magical creations, created by a spellcaster, with the intent in engaging in battle. 

#### Describe the solution
Added the MAGIC_FOCUS flag to the ethereal weapons. This is what the flag was created for after all.

#### Describe alternatives you've considered
Not doing anything.

### CODE REVIEW

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 1]
```diff
@@ -274,7 +294,8 @@
       "NO_REPAIR",
       "NO_SALVAGE",
       "NO_UNLOAD",
-      "NO_AMMO"
+      "NO_AMMO",
+      "MAGIC_FOCUS"
```
Comment (KorGgenT) [inline data/mods/Magiclysm/items/ethereal_items.json]: i'm not sure i want to make the woodbow even more powerful. it's already super easy to cast (low tier spell) and has incredible damage output.

Comment (foulman) [reply data/mods/Magiclysm/items/ethereal_items.json]: Oh, I wouldn't worry about it. now that we have the spell chaining I'm going to rework that and update it with the rest of the spells I'm building, put in a quiver w/arrows and all that, there's no other mechanic for me to use to make it more balanced.

Comment (foulman) [reply data/mods/Magiclysm/items/ethereal_items.json]: I agreed that doing a mana per shot would be a great way of handling it, but that's not possible rn, and I'm fine with limited shots. There was just no other way to make it work.

Comment (KorGgenT) [reply data/mods/Magiclysm/items/ethereal_items.json]: then it should probably get the change when that is updated, not be even more overpowered in the meantime.

**Pull request merged**